grpc/src/main/java/com/linecorp/armeria/server/grpc/StreamingServerCall.java

```diff
@@ -218,6 +218,8 @@ void doClose(Status status, Metadata metadata, boolean completed) {
             if (res.tryWrite(responseTrailers(ctx, status, metadata, trailersOnly))) {
                 res.close();
             }
+        } catch (RuntimeException e) {
```

Comment:
If an exception is raised, we need to create a new `StatusAndMetadata` from the exception. Otherwise, a call fails to write trailers but returns OK status.

Reply:
> How about catching Exception, like we do in UnaryServerCall

@trustin 
There's no checked-exception as of now, so I thought using RuntimeException was natural and narrow.

Reply:
A checked exception can be thrown unsafely with some trick like https://github.com/line/armeria/blob/5b384fbe27e7e6f9225d6db91cbb684d09dfbb5e/core/src/main/java/com/linecorp/armeria/common/util/Exceptions.java#L235